r-getting-started.Rmd

```diff
@@ -0,0 +1,329 @@
+Scrolling down the list, you'll note that some packages have the box on the
+left checked. This means the package has been loaded and is available for use.
+
+In addition to the built-in packages that come with every installation of R,
+we have the ability to use packages written by other programmers.
+One of the most common places to find such code is the
+[Comprehensive R Archive Network](https://cran.r-project.org), or CRAN.
+RStudio allows a straightforward way to download and install packages
+through CRAN. If you click on the "Install" button in the Packages tab,
+a window will appear allowing you to install from "Repository (CRAN)."
+In the space below "Packages," type "tidyverse." Making sure the box next to
+"Install dependencies" is checked, click "Install" to download and install
+this collection of code (which we'll be using in later sections).
+
+If you'd prefer to use code to install packages, rather than the RStudio pane,
+you could include the following in your script:
+
+```{r eval=FALSE}
+install.packages("tidyverse")
+library(tidyverse)
+```
+
+The first line is equivalent to clicking "Install," while the second line
+performs the same task as checking the box in the Packages pane.
+
+## Getting Help in RStudio {#r-getting-started-rstudio-help}
+
+There are a number of additional features in RStudio that may be of
+use, but we'll discuss them in later sections as the need arises.
+The remaining tab that will be useful to us now is "Help," located in the lower
+right pane.
+
+You can search for more information on functions in R by typing the name
+of the function, such as "round", into the search box in the Help pane.
+The text that appears in the window below can sometimes be extensive.
+You can search within this particular help page
+using the second search box that reads "Find in Topic."
+In this case, type "decimal" to find information about how the function
+determines the number of decimal points to print in the output.
+
+Luckily, R help documentation tends to be formatted very consistently. At the
+very top, you'll see the name of the function ("Round", which in this case
+represents a collection of related functions) followed by the name of its
+package inside curly brackets ("{base}", this is important because sometimes
+different packages have functions with the same names!). Below that, a short title
+indicates the purpose of the function. "Description" is a more extensive
+explanation that should help you figure out if the function is appropriate for
+your use. "Usage" provides information on how the function should be applied
+through code, and "Arguments" is like a legend for the code described in Usage.
+The last subheadings, "Details" through "References" and "Examples", should be
+self-explanatory.
+
+When you're writing code in your script, you can also access help documentation
+by prefacing a function with a question mark:
+
+```{r eval=FALSE}
+?sum
```

Comment:
Maybe also mention the use of `??` to do a global search of the given word/object.

Reply:
got it!